### ISSUE TITLE
question about the results

### ISSUE THREAD

**Issue by Cassiel8196** [Hoyyyaard/LSceneLLM#9]
I want to ask whether the indicators tested by me according to the weight file you published are normal. There are mainly two problems:
1. Is the result（XR-QA） of CIDEr in the paper amplified by 100 times?
2. The indicator ranges of meteor and rouge are opposite to those in the paper. 

<img width="726" alt="Image" src="https://github.com/user-attachments/assets/68465a1e-ed31-4a01-b23f-58f44afc90f7" />

**Comment by Hoyyyaard** [comment]
1. Yes.
2. Can you please provide the full log?

**Comment by Cassiel8196** [comment]
I didn't use distributed training. Due to limited hard disk space, I chose 1/3 of the test dataset for testing


2025-03-24 11:25:57,612 - XR-QA-Evaluation - INFO - Copy the Config file from cfgs/XR-QA-Evaluation.yaml to ./experiments/XR-QA-Evaluation/cfgs/test_Release_Test/config.yaml
2025-03-24 11:25:57,613 - XR-QA-Evaluation - INFO - args.config : cfgs/XR-QA-Evaluation.yaml
2025-03-24 11:25:57,613 - XR-QA-Evaluation - INFO - args.launcher : pytorch
2025-03-24 11:25:57,613 - XR-QA-Evaluation - INFO - args.local_rank : 0
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.num_workers : 1
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.seed : 0
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.deterministic : False
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.sync_bn : False
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.exp_name : test_Release_Test
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.start_ckpts : None
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.ckpts : /data/LSceneLLM/release/release.pth
2025-03-24 11:25:57,614 - XR-QA-Evaluation - INFO - args.val_freq : 1
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.resume : False
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.test : True
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.finetune_model : False
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.scratch_model : False
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.label_smoothing : False
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.mode : None
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.way : -1
2025-03-24 11:25:57,615 - XR-QA-Evaluation - INFO - args.shot : -1
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.fold : -1
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.visualization_attn : False
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.experiment_path : ./experiments/XR-QA-Evaluation/cfgs/test_Release_Test
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.tfboard_path : ./experiments/XR-QA-Evaluation/cfgs/TFBoard/test_Release_Test
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.log_name : XR-QA-Evaluation
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.use_gpu : True
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.distributed : True
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - args.world_size : 1
2025-03-24 11:25:57,616 - XR-QA-Evaluation - INFO - config.optimizer = edict()
2025-03-24 11:25:57,617 - XR-QA-Evaluation - INFO - config.optimizer.type : Adam
2025-03-24 11:25:57,617 - XR-QA-Evaluation - INFO - config.optimizer.kwargs = edict()
2025-03-24 11:25:57,617 - XR-QA-Evaluation - INFO - config.optimizer.kwargs.lr : 5e-06
2025-03-24 11:25:57,617 - XR-QA-Evaluation - INFO - config.optimizer.kwargs.weight_decay : 0.05
2025-03-24 11:25:57,617 - XR-QA-Evaluation - INFO - config.scheduler = edict()
2025-03-24 11:25:57,617 - XR-QA-Evaluation - INFO - config.scheduler.type : CosLR
2025-03-24 11:25:57,617 - XR-QA-Evaluation - INFO - config.scheduler.kwargs = edict()
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.scheduler.kwargs.epochs : 5
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.scheduler.kwargs.initial_epochs : 0.1
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.dataset = edict()
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.dataset.train = edict()
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.dataset.train._base_ = edict()
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.dataset.train._base_.NAME : HD_Hm3dQADataset
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.dataset.train._base_.N_POINTS : 40000
2025-03-24 11:25:57,618 - XR-QA-Evaluation - INFO - config.dataset.train._base_.NUM_GROUP : 128
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.GROUP_SIZE : 384
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.REGION_N_POINTS : 8192
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.REGION_NUM_GROUP : 128
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.REGION_GROUP_SIZE : 96
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.INSTANCE_N_POINTS : 1024
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.INSTANCE_NUM_GROUP : 128
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.INSTANCE_GROUP_SIZE : 16
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.PREPROCESS : False
2025-03-24 11:25:57,619 - XR-QA-Evaluation - INFO - config.dataset.train._base_.OPENSCENE : True
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.train.others = edict()
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.train.others.subset : train
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.train.others.npoints : 1024
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.train.others.whole : True
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.train.others.shuffle : True
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.train.others.bs : 1
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.train.others.tbs : 1
2025-03-24 11:25:57,620 - XR-QA-Evaluation - INFO - config.dataset.val = edict()
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_ = edict()
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.NAME : HD_Hm3dQADataset
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.N_POINTS : 40000
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.NUM_GROUP : 128
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.GROUP_SIZE : 384
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.REGION_N_POINTS : 8192
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.REGION_NUM_GROUP : 128
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.REGION_GROUP_SIZE : 96
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.INSTANCE_N_POINTS : 1024
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.INSTANCE_NUM_GROUP : 128
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.INSTANCE_GROUP_SIZE : 16
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.PREPROCESS : False
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val._base_.OPENSCENE : True
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val.others = edict()
2025-03-24 11:25:57,621 - XR-QA-Evaluation - INFO - config.dataset.val.others.subset : val
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.dataset.val.others.shuffle : False
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.dataset.val.others.bs : 1
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.dataset.val.others.tbs : 1
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model = edict()
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.NAME : Openscene
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.DTYPE : FP32
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.distributed : FSDP
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.FLEX : True
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.DENSE_TOKEN_NUM : 4
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.DENSE_TOKEN_SELECT_THRESHOLD : 96
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.quantization : False
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.START_FLEX_LAYER : 8
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.trans_dim : 768
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.use_openscene : True
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.N_POINTS : 40000
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.NUM_GROUP : 128
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.GROUP_SIZE : 384
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.REGION_N_POINTS : 8192
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.REGION_NUM_GROUP : 8
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.REGION_GROUP_SIZE : 1024
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.INSTANCE_N_POINTS : 1024
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.INSTANCE_NUM_GROUP : 1
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.INSTANCE_GROUP_SIZE : 1024
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.model.openscene_features_base_dir : /dataset/HM3D/OpenScene_Scan_Features/HM3D
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.finetune : False
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.total_bs : 1
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.step_per_update : 1
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.max_epoch : 5
2025-03-24 11:25:57,622 - XR-QA-Evaluation - INFO - config.consider_metric : CDL1
2025-03-24 11:25:57,623 - XR-QA-Evaluation - INFO - Distributed training: True
2025-03-24 11:25:57,623 - XR-QA-Evaluation - INFO - Set random seed to 0, deterministic: False
2025-03-24 11:25:58,697 - XR-QA-Evaluation - INFO - DENSE_TOKEN_SELECT_THRESHOLD is set to 96
2025-03-24 11:25:58,698 - XR-QA-Evaluation - INFO - START_FLEX_LAYER is set to 8
2025-03-24 11:25:58,698 - XR-QA-Evaluation - INFO - FLEX_WARM_UP is set to 20
2025-03-24 11:25:58,698 - XR-QA-Evaluation - INFO - SELECT_STRATEGY is set to NORMAL
2025-03-24 11:27:27,189 - XR-QA-Evaluation - INFO - [VALIDATION] Start validating epoch -1
2025-03-24 12:25:36,263 - XR-QA-Evaluation - INFO - 
----------------------Evaluation hd_scene_qa-----------------------

2025-03-24 12:25:36,264 - XR-QA-Evaluation - INFO - [BLEU-1] Mean: 0.4065, Max: 1.0000, Min: 0.0000
[BLEU-2] Mean: 0.3154, Max: 1.0000, Min: 0.0000
[BLEU-3] Mean: 0.2409, Max: 1.0000, Min: 0.0000
[BLEU-4] Mean: 0.1908, Max: 1.0000, Min: 0.0000
[CIDEr] Mean: 1.2603, Max: 10.0000, Min: 0.0000
[ROUGE-L] Mean: 0.3887, Max: 1.0000, Min: 0.0000
[METEOR] Mean: 0.2073, Max: 1.0000, Min: 0.0000

**Comment by Hoyyyaard** [comment]
It seems that the log is correct. Please evaluate the whole test set.

**Comment by Cassiel8196** [comment]
Thank you for your reply. I will evaluate the whole test set later. 

The Q-A task achieved only 28% accuracy in color recognition for objects. While my evaluation was limited to one-third of the dataset, the poor performance is unexpected. I want to ask whether the authors have validated the model's capability in visual grounding tasks.

**Comment by Beining1** [comment]
Thank you for your reply. I will evaluate the whole test set later. 

The Q&A task achieved only 28% accuracy in color recognition for objects. While my evaluation was limited to one-third of the dataset, the poor performance is unexpected. I would like to know whether the authors have validated the model's capability in visual grounding tasks.
